### PR TITLE
OVR-1578 error handling full-seo

### PR DESCRIPTION
**Pull request by JWMB**
https://humany.atlassian.net/browse/OVR-1578

### CODE REVIEW

File: SEO/tutorial-seo-full.md [hunk 1]
```diff
@@ -63,12 +63,18 @@ private string DownloadWidgetHtml(string widgetName, string basePath, string pat
...
   url.Query = queryString.ToString();
 
   // Download and return content as a string (using HTTP GET)
-  using (var client = new WebClient())
-    return client.DownloadString(url.ToString());
+  using (var client = new HttpClient())
+  {
+    var response = await client.GetAsync(ub.Uri);
+    // Note: regular widget will load even if we get e.g. a 404 here
+    // (could be b/c page hasn't been indexed yet)
```
Comment (laurilubi) [inline SEO/tutorial-seo-full.md]: Tydligare att skriva ut "because"?

File: SEO/tutorial-seo-full.md [hunk 2]
```diff
@@ -63,12 +63,18 @@ private string DownloadWidgetHtml(string widgetName, string basePath, string pat
 
   // Build the remote url and append the above query strings
   var myApplication = "seo-customer";
-  var url = new UriBuilder($"https://seo.humany.net/v2/{myApplication}/{widgetName}/path");
+  var url = new UriBuilder($"https://seo.humany.net/v2/{myApplication}/{widgetName}/{path}");
```
Comment (laurilubi) [inline SEO/tutorial-seo-full.md:66]: nice catch

File: SEO/tutorial-seo-full.md (hunk 1 above)
Comment (laurilubi) [reply SEO/tutorial-seo-full.md]: Man behöver catcha `HttpRequestException` enligt https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.getasync?view=netcore-3.0

Comment (JWMB) [reply SEO/tutorial-seo-full.md]: 👍

**Pull request merged**